virtual-desktop/src/app/plugin-manager/plugin-factory/iframe/iframe-plugin-factory.ts

```diff
@@ -75,8 +86,8 @@ export class IFramePluginFactory extends PluginFactory {
     return Promise.resolve();
   }
 
-  loadPlugin(pluginDefinition: MVDHosting.DesktopPluginDefinition): Promise<CompiledPlugin> {
-    const componentClass = this.createIFrameComponentClass(pluginDefinition);
+  loadPlugin(pluginDefinition: MVDHosting.DesktopPluginDefinition, pluginId: MVDHosting.ViewportId): Promise<CompiledPlugin> {
```

Comment:
You shouldn't call the viewportId pluginId, since pluginDefinition.identifier is to me, also pluginId and thus misleading.